axiom/nr_app_harvest.c

```diff
@@ -12,6 +12,19 @@
 #include "nr_app_harvest_private.h"
 #include "util_logging.h"
 
+/*
+ * The build pipeline for PHP 8.2 uses a Debian image that is built on a
+ * version of glibc that contains an newer glibc version symbol for `pow()`
+ * which is incompatible with older distributions, causing RPM installs to fail.
+ *
+ * To pin the version of `pow` to a version compatible with all NR PHP Agent
+ * supported OSes, we utilize `asm()` and `.symver` to instruct the linker to
+ * select an older version- in this case, 2.17.
+ */
+#if defined(__GLIBC__)
+__asm__(".symver pow,pow@GLIBC_2.17");
+#endif
```

Comment:
Extending range of @mfulb's [suggestion](https://github.com/newrelic/newrelic-php-agent/pull/624/files#r1120806448) so that it can be applied from GitHub's web UI:
```suggestion
#if defined(__GLIBC__)
#if ((__GLIBC__) >= 2) && ((__GLIBC_MINOR__) > 17)
__asm__(".symver pow,pow@GLIBC_2.17");
#endif
#endif
```